v3.5.0 : Link error on Ubuntu 20.04

It seems that the OpenGL library is not being included during the link phase

```
[ 28%] Linking CXX executable ../../bin/glViewer
cd /home/nyue/projects/OpenSubdiv/3.5.0/build/examples/glViewer && /home/nyue/systems/cmake/3.23.2/bin/cmake -E cmake_link_script CMakeFiles/glViewer.dir/link.txt --verbose=1
/usr/bin/c++ -O3 -DNDEBUG  -fopenmp CMakeFiles/glViewer.dir/glViewer.cpp.o ../../regression/common/CMakeFiles/regression_common_obj.dir/arg_utils.cpp.o ../../regression/common/CMakeFiles/regression_common_obj.dir/shape_utils.cpp.o ../../regression/common/CMakeFiles/regression_far_utils_obj.dir/far_utils.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/font_image.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/hdr_reader.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/hud.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/objAnim.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/patchColors.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/viewerArgsUtils.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/glControlMeshDisplay.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/glHud.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/glUtils.cpp.o ../common/CMakeFiles/examples_common_gl_obj.dir/glShaderCache.cpp.o -o ../../bin/glViewer   -L/home/nyue/systems/tbb/tbb44_20160526oss/lib/intel64/gcc4.4  -Wl,-rpath,/home/nyue/projects/OpenSubdiv/3.5.0/build/lib:/home/nyue/systems/tbb/tbb44_20160526oss/lib/intel64/gcc4.4: ../../lib/libosdGPU.so.3.5.0 ../../lib/libosdCPU.so.3.5.0 /usr/lib/x86_64-linux-gnu/libGLEW.so -ldl /usr/lib/x86_64-linux-gnu/libglfw.so /usr/lib/x86_64-linux-gnu/libXrandr.so /usr/lib/x86_64-linux-gnu/libXxf86vm.so /usr/lib/x86_64-linux-gnu/libXcursor.so /usr/lib/x86_64-linux-gnu/libXinerama.so /usr/lib/x86_64-linux-gnu/libXi.so /usr/lib/x86_64-linux-gnu/libSM.so /usr/lib/x86_64-linux-gnu/libICE.so /usr/lib/x86_64-linux-gnu/libX11.so /usr/lib/x86_64-linux-gnu/libXext.so -lpthread -lrt -ldl -lgomp -ltbb -ltbb_debug 
/usr/bin/ld: CMakeFiles/glViewer.dir/glViewer.cpp.o: undefined reference to symbol 'glEnable'
/usr/bin/ld: /lib/x86_64-linux-gnu/libGL.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [examples/glViewer/CMakeFiles/glViewer.dir/build.make:148: bin/glViewer] Error 1
make[2]: Leaving directory '/home/nyue/projects/OpenSubdiv/3.5.0/build'
make[1]: *** [CMakeFiles/Makefile2:3841: examples/glViewer/CMakeFiles/glViewer.dir/all] Error 2
make[1]: Leaving directory '/home/nyue/projects/OpenSubdiv/3.5.0/build'
make: *** [Makefile:146: all] Error 2
```

My configuration options are these
```
export TBB_ROOT=/home/nyue/systems/tbb/tbb44_20160526oss
cmake \
    -D OpenGL_GL_PREFERENCE=GLVND \
    -D PTEX_LOCATION=/home/nyue/systems/ptex/2.4.2 \
    -D Python_EXECUTABLE=/home/nyue/projects/OpenSubdiv/3.5.0/dev_venv/bin/python \
    -D CMAKE_BUILD_TYPE=Release \
    -D CMAKE_INSTALL_PREFIX=$HOME/systems/OpenSubdiv/3.5.0 \
    -D NO_CLEW=ON \
    -D NO_CUDA=ON \
    -D NO_DOC=OFF \
    -D NO_DX=ON \
    -D NO_EXAMPLES=OFF \
    -D NO_GLEW=OFF \
    -D NO_GLFW=OFF \
    -D NO_GLFW_X11=OFF \
    -D NO_GLTESTS=OFF \
    -D NO_LIB=OFF \
    -D NO_MACOS_FRAMEWORK=ON \
    -D NO_METAL=ON \
    -D NO_OMP=OFF \
    -D NO_OPENCL=ON \
    -D NO_OPENGL=OFF \
    -D NO_PTEX=OFF \
    -D NO_REGRESSION=ON \
    -D NO_TBB=OFF \
    -D NO_TESTS=OFF \
    -D NO_TUTORIALS=OFF \
    ../OpenSubdiv-3_5_0
```

I am building on a machine that does not have a discrete graphics card/GPU

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -490,17 +490,16 @@ if(GLEW_FOUND AND GLEW_INCLUDE_DIR)
         ${PROJECT_SOURCE_DIR}/glLoader)
     set(OPENGL_LOADER_LIBRARIES
         ${GLEW_LIBRARY}
-        ${OPENGL_gl_LIBRARY}
+        OpenGL::GL
         ${CMAKE_DL_LIBS})
 
 elseif(OPENGL_FOUND)
 
     osd_detect_gl_version(${PROJECT_SOURCE_DIR}/glLoader/glApi.h)
     set(OPENGL_LOADER_INCLUDE_DIRS
-        ${OPENGL_INCLUDE_DIR}
         ${PROJECT_SOURCE_DIR}/glLoader)
     set(OPENGL_LOADER_LIBRARIES
-        ${OPENGL_gl_LIBRARY}
+        OpenGL::GL
         ${CMAKE_DL_LIBS})
 
 endif()

